openhtf/io/frontend/__main__.py

```diff
@@ -145,49 +141,29 @@ def _Discover(self):
                                FLAGS.discovery_port,
                                FLAGS.discovery_ttl)
     for host, response in responses:
+      port = None
       try:
         port = json.loads(response)[PING_RESPONSE_KEY]
         self._Track(host, port)
```

Comment:
you might consider `int(port)` here to fail as fast as possible if you get a weird value in the response.